paramedic-plugin/paramedic.js

```diff
@@ -0,0 +1,74 @@
+var io = cordova.require('cordova-plugin-paramedic.socket.io');
```

Comment:
This is cordovaRequire (paramedic.js is plugin script, not a node module ) which works slightly different.
The _require("socket.io")_  and _require("./socket.io")_ won't work in this case.